src/UnityMvvmToolkit.UnityPackage/Assets/Plugins/UnityMvvmToolkit/Runtime/UITK/BindableUIElements/BindableDropdownField.cs

```diff
@@ -0,0 +1,179 @@
+                _itemsSource.Value.CollectionChanged += OnItemsCollectionChanged;
+                choices = new List<string>(_itemsSource.Value);
+            }
+            
+            if (string.IsNullOrWhiteSpace(BindingSelectedItemPath) == false)
+            {
+                _selectedItemBindingData ??= BindingSelectedItemPath.ToPropertyBindingData();
+                _selectedItemProperty = objectProvider.RentProperty<string>(context, _selectedItemBindingData);
+                _selectedItemProperty.ValueChanged += OnSelectedItemValueChanged;
+                
+                var isContains = choices.Contains(_selectedItemProperty.Value);
+                if (isContains == true)
+                {
+                    UpdateControlValue(_selectedItemProperty.Value);
+                }
+                
+                this.RegisterValueChangedCallback(OnControlValueChanged);
+                _selectedItemProperty.Value = choices.Count > 0 ? choices[0] : default;
+            }
+        }
+
+        private void OnItemsCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+        {
+            switch (e.Action)
+            {
+                case NotifyCollectionChangedAction.Add:
+                    
+                    foreach (string newItem in e.NewItems)
+                    {
+                        choices.Add(newItem);
+                    }
+                    
+                    break;
+                
+                case NotifyCollectionChangedAction.Remove:
+
+                    if (e.OldStartingIndex < 0)
+                    {
+                        throw new InvalidOperationException("RemovedItemNotFound");
+                    }
+                    
+                    foreach (string oldItem in e.OldItems)
+                    {
+                        choices.Remove(oldItem);
+                    }
+
+                    break;
+                
+                case NotifyCollectionChangedAction.Replace:
+                    
+                    if (e.NewItems.Count != 1 || e.OldItems.Count != 1)
+                    {
+                        throw new NotSupportedException("RangeActionsNotSupported");
+                    }
+                    
+                    if (e.NewItems?[0] is string replacingValue &&
+                        e.OldItems?[0] is string replacedValue)
+                    {
+                        int indexReplacedValue = choices.FindIndex(s => s == replacedValue);
+
+                        if (indexReplacedValue != -1)
+                        {
+                            choices[indexReplacedValue] = replacingValue;
+                        }
+                    }
+                    break;
+                
+                case NotifyCollectionChangedAction.Move:
+                    
+                    if (e.NewItems.Count != 1)
+                    {
+                        throw new NotSupportedException("RangeActionsNotSupported");
```

Comment:
Why didn't you implement the same range action support as for `Add` and `Remove`?